### ISSUE TITLE
[a11y related feature request] Allow divs instead of buttons for pivot items

### ISSUE THREAD

**Issue by solidprinciples** [microsoft/fluentui#19240]
#### Describe the feature that you would like added

The current implementation for PivotItems use Button tags, which presents a problem with using Links <a> inside - where it is an a11y issue that it has a focusable element inside a focusable element. One way to alleviate this would be to allow for overriding the main render of the button potentially as a div or otherwise, or giving an option to just render child content.

#### What component or utility would this be added to

Pivot

#### Have you discussed this feature with our team, and if so, who

Somewhat - messaged @behowell

**Comment by smhigley** [comment]
@solidprinciples could you provide an example of what use case you're trying to create? Links are not allowed inside tab items regardless of whether they're made with a `<button role="tab">` or a `<div role="tab">`. The only difference is whether the browser does some error correction, or whether it fails silently for screen reader users.

Depending on what you're trying to create, it sounds like you might need another type of control entirely.
